content/en/docs/setup/upgrade/steps/index.md

```diff
@@ -58,14 +49,14 @@ This is suitable for cases where `kubectl apply` was used to deploy Istio CNI.
     $ NAMESPACE=$(kubectl get pods -l k8s-app=istio-cni-node --all-namespaces --output='jsonpath={.items[0].metadata.namespace}')
     {{< /text >}}
 
-1. If `istio-cni` is currently installed in a namespace other than `kube-system`
+2. If `istio-cni` is currently installed in a namespace other than `kube-system`
```

Comment:
Please leave numbered lists with 1. The right numbers will be inserted automatically when the page is generated.